webview/src/experiments/components/table/styles.module.scss

```diff
@@ -450,19 +454,58 @@ $workspace-row-edge-margin: $edge-padding - $cell-padding;
         align-items: center;
         position: relative;
         left: -$cell-padding;
+        height: 100%;
 
         .starSwitch {
           display: inline-flex;
           align-items: center;
+          justify-content: center;
           width: 14px;
-          margin-left: 8px;
           opacity: 0.4;
           cursor: pointer;
 
           &[aria-checked='true'] {
             opacity: 1;
           }
         }
+
+        .indicatorIcon {
+          display: inline-flex;
+          align-items: center;
+          justify-content: center;
+          width: 1.3rem;
+          height: 100%;
+          padding: 0;
+          margin: 0;
+        }
+
+        .indicatorCount {
+          z-index: 2;
+          &[title='0'] {
+            display: none;
+          }
+        }
+
+        .bullet {
+          flex: 0 0 14px;
+          width: 14px;
+          &::after {
+            border: none;
+          }
+
+          &.subRunsCount {
```

Comment:
If we are going to push the contents of the first column right to make space for this indicator then we need to increase the min width of the first column:

<img width="1211" alt="image" src="https://user-images.githubusercontent.com/37993418/178473015-5537cf7e-0ffb-4b08-a072-8055db6de282.png">

Reply:
picture is from Storybook snapshot (names are completely hidden)